dhcpv6/dhcpv6relay.go

```diff
@@ -3,6 +3,7 @@ package dhcpv6
 import (
 	"errors"
 	"fmt"
+	"github.com/insomniacslk/dhcp/iana"
```

Comment:
Better to keep it consistent with other files and have this import in the 2nd block
https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/dhcpv6.go#L8